spec/providers/coreIntegration/udpsocket.integration.src.js

```diff
@@ -27,14 +27,13 @@ module.exports = function(provider, setup) {
       return did.bind(null, markTask(name));
     };
 
-    var LOCALHOST = '127.0.0.1';
+    var LOCALHOST_V4 = '127.0.0.1';
     var checkSocketInfo = function(socketToCheck, port) {
       var getInfoTask = markTask('getInfo');
       socketToCheck.getInfo(function(state) {
-        expect(state).toEqual(jasmine.objectContaining({
-          'localAddress': LOCALHOST,
-          'localPort': port
-        }));
+        // On Chrome, this is "127.0.0.1".  On Firefox it's "localhost".
```

Comment:
Is there anyway we can rewrite our tests to use the subset of the jasmine
API that is common in both 1.7 and 2.0?

On Tue, Jan 6, 2015 at 10:05 AM, Benjamin M. Schwartz <
notifications@github.com> wrote:

> In spec/providers/coreIntegration/udpsocket.integration.src.js
> <https://github.com/freedomjs/freedom/pull/185#discussion-diff-22526675>:
>
> >      var checkSocketInfo = function(socketToCheck, port) {
> >        var getInfoTask = markTask('getInfo');
> >        socketToCheck.getInfo(function(state) {
> > -        expect(state).toEqual(jasmine.objectContaining({
> > -          'localAddress': LOCALHOST,
> > -          'localPort': port
> > -        }));
> > +        // On Chrome, this is "127.0.0.1".  On Firefox it's "localhost".
>
> @willscott <https://github.com/willscott> It turns out this test doesn't
> work on node, because freedom-for-node is stuck on jasmine-1.14, and this
> test uses some jasmine-2.0 features (e.g. spy.and.callFake instead of
> spy.andCallFake). So I don't know what it is on node ... and I'm not really
> sure how to proceed there, or with this pull request. Suggestions welcome.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freedomjs/freedom/pull/185/files#r22526675>.
>